### PR TITLE
[WIP]feat: provide a simple omise integration mechanism

### LINKED ISSUE #5823: Implement  Omise Payment Gateway
For the OpenTechSummit in Thailand we need local payment gateways. The most popular one is omise. Please implement it.

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #5823 

#### Short description of what this resolves:
Try and implement a sample omise transaction endpoint

### CODE REVIEW

**Comment by uds5501** [comment]
@CosmicCoder96 I took the liberty to try and implement a sample end point on my end. If you have any progress made already, then I shall close it, otherwise you may use it as a reference. 
Ps: It is crude and filled with test stubs at the moment, please ignore that.

Also, I shall be detailing certain complications which I came across during the implementation which I would mention here and like to discuss with you, @iamareebjamal , @mariobehling  in the evening

File: app/api/orders.py [hunk 1]
```diff
@@ -417,3 +417,64 @@ def create_paypal_payment(order_identifier):
...
+            )
+            if charge.failure_code is not None:
+                print('Yup, some error {}'.format(charge.failure_message))
+            else:
+                flash("Card successfully charged!  Order ID: {}".format(order_id))
+        # For NON CARD transfers
+        elif token == '' and source != '':
+            charge = omise.Charge.create(
+                amount=int(myorder.amount) * 100,
+                currency='THB',
```
Comment (kushthedude) [inline app/api/orders.py]: Why are setting currency as `THB`

File: app/api/orders.py [hunk 2]
```diff
@@ -417,3 +417,64 @@ def create_paypal_payment(order_identifier):
...
+    # TEST STUBS
+
+    print('you have reached omise transactions\n')
+    print(request.form)
+    token = request.form.get('omiseToken')
+    source = request.form.get('omiseSource')
+
+    print('\n', token, source)
+
+    omise.api_secret = 'skey_test_xxxxxxxxxxxx'
```
Comment (mrsaicharan1) [inline app/api/orders.py]: @uds5501 Has this been working locally? Also, would be great if you could fetch the omise credentials from the db, similar to stripe and PayPal
```
class StripeAuthorization(SoftDeletionModel):
    """
    Stripe authorization information for an event.
    """
    __tablename__ = 'stripe_authorizations'

    id = db.Column(db.Integer, primary_key=True)
    stripe_secret_key = db.Column(db.String)
    stripe_refresh_token = db.Column(db.String)
    stripe_publishable_key = db.Column(db.String)
    stripe_user_id = db.Column(db.String)
    stripe_auth_code = db.Column(db.String)
```

**Comment by uds5501** [comment]
@mrsaicharan1 Follow the last 2 commits of this pr : https://github.com/uds5501/open-event-frontend/tree/omise-mech-front

to setup the frontend and test it accordingly

**Comment by mrsaicharan1** [comment]
Yeah, I'll work on this.

**Pull request closed without merging**